OxfordBodleian/Juel-Jensen/BDLaethg34.xml

```diff
@@ -160,25 +118,671 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                </handNote>
+                            </handDesc>
+                            
+                            <additions>
+                                <list>
+                                    <item xml:id="e5">
+                                        <desc>Occasionally, omitted characters have been added above the line (e.g. <locus target="#37r #48v"/>); omitted words are added in 
+                                            the margins  (e.g., with use of <foreign xml:lang="gez">tamallas</foreign>-sign, <locus target="#56v #57r"/>); corrections are written above
+                                        erasures (e.g. <locus target="#38r"/>).</desc>
+                                    </item>
+                                    
+                                    <item xml:id="e6">
+                                        <locus target="#45r #55r"/>
+                                        <desc>Note on the liturgical reading in the upper margin.</desc>
+                                        <q xml:lang="gez">ዘመንፈቀ፡ ሌሊት፡</q>
+                                    </item>
+                                    
+                                    <item xml:id="e7">
+                                        <locus target="#57r"/>
+                                        <desc>Note on the liturgical reading in the upper margin.</desc>
+                                        <q xml:lang="gez">ዘሰርክ፡</q>
+                                    </item>
+                                    
+                                    <item xml:id="e8">
+                                        <locus target="#73v #74r"/>
+                                        <desc>Pen trials in black and red.</desc>
+                                        <q xml:lang="gez"/>
+                                    </item>
+                                    
+                                    <item xml:id="e9">
+                                        <locus target="#74r"/>
+                                        <desc>Note mentioning <persName ref="PRS3430DawitII"/> and <persName ref="PRS6777Marqos"/>.</desc>
+                                        <q xml:lang="gez">ዘወጽአ፡ በዘመኑ፡ ንጉስነ፡ <persName ref="PRS3430DawitII">ዳዊት፡</persName> 
+                                            <sic resp="DR">ወጣጣስነ፡</sic> <persName ref="PRS6777Marqos">አባማርቆስ፡</persName> ፫ እትሮ 
+                                            <persName ref="PRS13689GabraD">ግብረ ድንግል</persName></q>
+                                    </item>
+                                </list>
+                            </additions>
+                            
+                        </physDesc>
+                        
+                        <history>
+                            <origin>
+                                <origDate notBefore="1700" notAfter="1899" evidence="lettering"/>
```

Comment:
I think your reconstruction of the 1716 dating is correct. So, I would replace 1899 with 1716 as the latest possible date of production and change "lettering" with "prosopography".